app/decorators/mfa_context.rb

```diff
@@ -53,6 +53,16 @@ def two_factor_configurations
       [piv_cac_configuration, auth_app_configuration]
   end
 
+  # rubocop:disable Metrics/AbcSize
+  def enabled_mfa_methods_count
```

Comment:
nice!

Reply:
I guess we don't need this: backup_code_configurations.unused.any?

Reply:
The `backup_code_configurations` method already adds the `unused` scope

Reply:
Ah.  I see.